src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,28 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+
+    private final String tagAction;
```

Comment:
think why `String` may not be the best data type for this particular field? 
what would be more appropriate?